src/components/bulk-email-tool/bulk-email-form/bulkEmailForm.scss

```diff
@@ -0,0 +1,10 @@
+// Flip a leading icon to be a trailing icon
+.send-email-btn {
+    > span {
+        flex-direction: row-reverse;
+        gap: 0.5rem;
+        > span {
+            margin: 0;
+        }
+    }
+}
```

Comment:
mega-nit: we usually put new lines at the ends of files. Im not sure if this is caught by linting 🤔